### ISSUE TITLE
[Suggestion]Redefining the relationship between tools and materials

### ISSUE THREAD

**Issue by MokoBlack** [Sirse/MineFantasy-III#11]
Have you ever thought that MF forging is no longer a simple upgrade material for equipment?
It's no longer like upgrading advanced materials to increase weapon damage.It's just that all the materials that have reached a certain level have their strengths and weaknesses.
For example, a kind of metal is soft and light. It can be used as a short sword. Its advantages are easy production, fast attack speed and puncture as the main attack.
In short, materials will make a weapon's durability, attack speed, type of damage（maby attack type too）, weight, and shape vary in a variety of ways, rather than simply better materials - higher damage. Tools and armor can also consider their performance similarly.
This is an attempt to break away from the traditional relationship between vanilla tools and materials.
Also want to get some hard core like
